Example/PaymentSheet Example/PaymentSheet Example/AppearancePlaygroundView.swift

```diff
@@ -127,6 +127,16 @@ struct AppearancePlaygroundView: View {
             set: { self.appearance.primaryButton.backgroundColor = UIColor($0) }
         )
 
+        let primaryButtonSuccessColorBinding = Binding(
+            get: { Color(self.appearance.primaryButton.successBackgroundColor) },
+            set: { self.appearance.primaryButton.successBackgroundColor = UIColor($0) }
+        )
+
+        let primaryButtonSuccessTextColorBinding = Binding(
+            get: { Color(self.appearance.primaryButton.successTextColor ?? self.appearance.primaryButton.textColor ?? .white) },
```

Comment:
Just a non-blocking comment - IIUC seems like it'd be more accurate to refactor our code/UI to allow for nil values here instead of assuming what the default color is.  If we ever change the defaulting logic this will be wrong.

Reply:
Agree that would be ideal, always resolving to a non-nil color is mainly driven by the fact that `ColorPicker` requires a non-nil color. Maybe a button with text `nil` that when tapped presents the color picker? I looked around at this a bit but programmatically presenting a `ColorPicker` from a different button seems non-trivial: https://github.com/franklynw/PresentableColorPicker

Reply:
I guess there is another way, slightly hacky: https://stackoverflow.com/questions/73382095/activate-color-picker-overlay-when-button-pressed-swiftui